modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java

```diff
@@ -84,30 +84,20 @@
 @Component(
 	property = "content.type=application/xliff+xml",
 	service = {
-		TranslationInfoItemFieldValuesImporter.class,
-		TranslationSnapshotProvider.class
+		TranslationInfoItemFieldValuesImporter.class
 	}
 )
 public class XLIFFInfoFormTranslationImporter
-	implements TranslationInfoItemFieldValuesImporter,
-			   TranslationSnapshotProvider {
-
-	@Override
-	public TranslationSnapshot getTranslationSnapshot(
-			long groupId, InfoItemReference infoItemReference,
-			InputStream inputStream)
-		throws IOException, PortalException {
-
-		return _getTranslationSnapshot(
-			groupId, infoItemReference, inputStream, true);
-	}
+	implements TranslationInfoItemFieldValuesImporter{
 
 	@Override
 	public InfoItemFieldValues importInfoItemFieldValues(
 			long groupId, InfoItemReference infoItemReference,
 			InputStream inputStream)
 		throws IOException, XLIFFFileException {
 
+
+
 		TranslationSnapshot translationSnapshot = _getTranslationSnapshot(
```

Comment:
Here have erro, because in class `TranslationSnapshot`, he isn't find the method `_getTranslationSnapshot`